### ISSUE TITLE
[DEPRECATION v22.6] Setting the ErrorHandler fallback value directly is deprecated and no longer supported.

### ISSUE THREAD

**Issue by harpener** [ashleysommer/sanic-cors#57]
Hi guys,

I am getting a following deprecation warning:
```
DeprecationWarning: [DEPRECATION v22.6] Setting the ErrorHandler fallback value directly is deprecated and no longer supported.
This feature will be removed in v22.6.
Instead, use app.config.FALLBACK_ERROR_FORMAT.
```

From what I found, it is caused by this: https://github.com/ashleysommer/sanic-cors/blob/2.0.0/sanic_cors/extension.py#L406

I suggest to remove the parameter (the default value seems to correspond to `"auto"` anyway).

Can you please take a look?

Thanks

**Comment by liuyuanjun520** [comment]
+1

**Comment by ashleysommer** [comment]
Thanks for the bug report. I'll release a fix for this before Sanic v22.3 is released in March.

**Comment by ashleysommer** [comment]
Sanic-CORS v2.0.1 released today, with this issue fixed.
